migrations/Version20200719160409.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200719160409 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $this->addSql(<<<EOF
+CREATE TABLE candidate_function (
+    id INT NOT NULL,
+    label VARCHAR(64) NOT NULL,
+    max_count INT DEFAULT NULL,
+    `show` TINYINT(1) NOT NULL,
+    `order` SMALLINT NOT NULL,
+    UNIQUE INDEX UNIQ_6BF054EEEA750E8 (label),
+    PRIMARY KEY(id)
+) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB
+EOF);
+
+        $this->addSql(<<<EOF
+CREATE TABLE candidate (
+    id INT NOT NULL,
+    function_id INT NOT NULL,
+    candidate_with INT DEFAULT NULL,
+    person_id INT NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    INDEX IDX_C8B28E4467048801 (function_id),
+    UNIQUE INDEX UNIQ_C8B28E44A0B9DAAC (candidate_with),
+    PRIMARY KEY(id)
+) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB
+EOF);
+
+        $this->addSql('ALTER TABLE candidate ADD CONSTRAINT FK_C8B28E4467048801 FOREIGN KEY (function_id) REFERENCES candidate_function (id)');
+        $this->addSql('ALTER TABLE candidate ADD CONSTRAINT FK_C8B28E44A0B9DAAC FOREIGN KEY (candidate_with) REFERENCES candidate (id)');
+
+        $this->addSql(<<<EOF
+INSERT INTO `candidate_function` (`id`, `label`, `max_count`, `show`, `order`) VALUES
+(10,	'náčelník',	1,	1,	10),
```

Comment:
Proč má id takovou řadu, to je nějaké skautIS id?
Co znamenají max_count, show a order? Nedají se vytáhnout odněkud ze skautIS?

Reply:
Jsou to id ze skautisu. Načítání ze skautisu mi přišlo tehdy zbytečně komplikované kdyz presne vim ktere pozice chci a budu to neustále vypisovat.

Reply:
Potom bych asi použil něco jako functionId, id jinak používáme pro automaticky generovaná id

U max count, show a order předpokládám, že mají význam hlavně pro zobrazení

Reply:
já bych tam zachoval, ze ten objekt má nějaké id. asi je to mozne i jak navrhujes, ale klidne bych to nechal takto, pokud nemas silnou preferenci to měnit. 

max_count, show, order - ty jsou pouze v nasi aplikaci pro dalsi logiku - vyhodnocení voleb, zobrazení apod